src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -156,6 +156,12 @@ object Arisa : ConfigSpec() {
                 null,
                 description = "A tag used to indicate that Arisa should keep the ticket Awaiting Response"
             )
+            val softARPeriod by optional<Long?>(
+                null,
+                description = "The ticket can only be reopened by comments posted by people other than the reporter " +
```

Comment:
```suggestion
                description = "The ticket can also be reopened by comments posted by people other than the reporter " +
```
"Only" would mean that the reporter is not able to reopen it that way.